src/main/resources/application.properties

```diff
@@ -30,11 +30,11 @@ proxy.enabled=false
 proxy.host=noproxy
 proxy.port=3128
 
-timeouts.audit.read-ms=2000
+timeouts.audit.read-ms=10000
 timeouts.audit.connect-ms=1000
-timeouts.hmrc-api.read-ms=2000
+timeouts.hmrc-api.read-ms=100000
```

Comment:
This is for hmrc-service to the real hmrc endpoint - I think 10s would be fine for this.  We need 100s from ip-api to hmrc-service because that contains the name matching logic.

Reply:
Yes that makes sense. Resolved.